### ISSUE TITLE
Test for "symmetry divergence" during training

### ISSUE THREAD

**Issue by brilee** [tensorflow/minigo#84]
One of the symptoms of value net overfitting that we found was that the 8 board symmetries would yield wildly different results when put through the value net. So instead of the value net returning roughly consistent estimations, stdev=0.1 or so, it would instead disagree completely, with 7 symmetries saying that B+ 0.99, and the 8th symmetry saying W+ 0.99.

It'd be nice to log the average stdev of the 8 symmetries (or some other characterization of how divergent the symmetries are.)

**Comment by amj** [comment]
So a script to analyze these is in #139.  @sethtroisi do you think this makes sense as a metric to watch in TB, or should we put this in cloudygo?
